avm/res/event-grid/namespace/README.md

```diff
@@ -821,7 +821,7 @@ module namespace 'br/public:avm/res/event-grid/namespace:<version>' = {
       dynamic: [
         {
           key: 'dynamic1'
-          value: '<value>'
+          value: '\${client.authenticationName}'
```

Comment:
just double checking: is this intended as consequence of the utility update?

Reply:
Yes. It was incorrect before